skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -26,8 +30,10 @@ export default class search extends Component {
       Flashon: false,
       validScore: true,
       validCompetition: true,
-      scoreEntered:false
+      scoreEntered:false,
+      iconhouseUrl : '../components/assets/grayFlashOff.png'
```

Comment:
Other icons were imported please do the same here

Reply:
removed it